pkgs/tools/misc/flashrom/default.nix

```diff
@@ -18,6 +18,7 @@ stdenv.mkDerivation rec {
     sha256 = "0ax4kqnh7kd3z120ypgp73qy1knz47l6qxsqzrfkd97mh5cdky71";
   };
 
+  mesonFlags = lib.optional stdenv.isAarch64 [ "-Dpciutils=false" ];
```

Comment:
Nitpick: I think this is better either as `optionals` or without the brackets.

Reply:
Thanks for the hint, I will use lib.optionals next time, but since this is already merged I think this cosmetical detail is not worth another commit.